eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -83,10 +83,10 @@ def __str__(self):
             ' ClinVar records with allowed clinical significance, valid rs id and ' +
             'Variant->ENSG mapping were skipped due to a lack of EFO mapping (see ' +
             config.UNMAPPED_TRAITS_FILE_NAME + ').',
-            str(self.counters["n_records_no_recognised_allele_origin"]) +
-            ' ClinVar records with allowed clinical significance, ' +
-            'valid rs id, valid Variant->ENSG' +
-            ' mapping and valid EFO mapping were skipped due to a lack of a valid alleleOrigin.',
+            # str(self.counters["n_records_no_recognised_allele_origin"]) +
+            # ' ClinVar records with allowed clinical significance, ' +
+            # 'valid rs id, valid Variant->ENSG' +
+            # ' mapping and valid EFO mapping were skipped due to a lack of a valid alleleOrigin.',
```

Comment:
I think it is still useful to have this metric? It doesn't take much effort to calculate it and it is convenient for future reports.